extra/drive-strings/drive-strings.factor

```diff
@@ -0,0 +1,9 @@
+! Copyright (C) 2019 Alexander Ilin.
+! See http://factorcode.org/license.txt for BSD license.
+USING: byte-arrays io.encodings.string io.encodings.utf16
+kernel math sequences splitting windows.kernel32 ;
+IN: drive-strings
```

Comment:
I'm not sure about this vocab name, or if it shouldn't be part under ``windows`` vocabulary...

Reply:
Good idea, I'll move it under `windows.`.

Reply:
Ah, you've already merged it.

Reply:
It's all good -- being in ``extra`` makes me stress less about it.

Reply:
I could move it to ``windows.drive-arrays`` if you think that's better.

Reply:
Yes, it totally makes sense. There are no drive letters on other platforms, and this is specifically a demo/wrapper for a WinAPI function.
I should have treated it same as the `windows.version` vocab here.

Reply:
Except I don't really like `drive-arrays`, better `drive-strings` or simply `drives`. But I leave it up to you.

Reply:
I meant ``windows.drive-strings``, that was a typo!

Moved in 1e81dbdf17281dfcacaad7a1e685b3cdaf147fd3.

Reply:
You mean "copied", not "moved" : ))

Reply:
Oh geez.  Maybe my brain is on lockdown too.

Non-atomic move completed in a89474786edba391abd6da6e5d3ec9e7a4daacb6.